### ISSUE TITLE
Not getting a good acc

### ISSUE THREAD

**Issue by leochli** [shicai/MobileNet-Caffe#23]
Hi @shicai ,
I tried your depoly prototxt, yet didn't get a good acc result.

I'd be appreciated if you could share your solver prototxt for me to check.

thanks!

**Comment by shicai** [comment]
what is your result?
solver prototxt file has no effect on the val accuracy of the pretrained model.

**Comment by leochli** [comment]
Hi @shicai 
I only got 54 as top 1 acc and 79 as top 5 acc. Btw I didn't use the pretrained model, I'm trying to train the net from the imagenet dataset.
thanks!

**Comment by shicai** [comment]
training models on ImageNet is somehow very tricky.
you'd better train some simple models first.

**Comment by Coderx7** [comment]
@Shicai
First of all Thanks alot. 
It would be very helpful if you could share the training/testing script as well for replicating the exact results.

**Comment by leochli** [comment]
@shicai 
Hi shicai,

I've tuned the top 1 acc to 67%, still sort of a long way from the paper acc. I trained with batch size 256, 90 epochs and decay the lr rate by 0.1 every 30 epochs.

I'd be appreciated if there're any tricks that you could share with me to reach the 70 acc?

thanks a lot

**Comment by shicai** [comment]
please see: https://github.com/shicai/MobileNet-Caffe/issues/1
